zipline/assets/roll_finder.py

```diff
@@ -25,7 +25,7 @@ class RollFinder(with_metaclass(ABCMeta, object)):
     def _active_contract(self, oc, front, back, dt):
         raise NotImplementedError
 
-    def get_contract_center(self, root_symbol, dt, offset):
+    def _get_contract_center(self, root_symbol, dt, offset):
```

Comment:
So it seems like `_get_contract_center` and `get_contract_center` now mean different things. `_get_contract_center` returns what we would choose as the active contract with just the info we would have on `dt`, while `get_contract_center` returns the center given the calculated rolls (where in the base class, these are equivalent). Maybe we could rename `_get_contract_center` to make this clearer? To me the inheritance also feels a little convoluted otherwise.